src/parser/AdminSentences.h

```diff
@@ -223,6 +223,9 @@ class CreateSpaceSentence final : public Sentence {
     }
 
     std::vector<SpaceOptItem*> getOpts() {
+        if (spaceOpts_ == nullptr) {
+            return {};
+        }
```

Comment:
when create graph space without options, CreateSpaceSentence::toString function need to  deal with

Reply:
roger that!